### DISCUSSION TITLE
Multitouch support for desktop

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#17779] [Ideas] [answered]
Lazer does not currently have multitouch support on desktop, I tried to find any other issues posted but there seem to be none so I will create one. The system simply needs to be like stable, most recent tap location is the location of the cursor, if the number of fingers on the screen increases, click starting with M1, then if M1 is already clicked, swap to M2, if M2 is clicked, swap to M1.

**Comment by smoogipoo** [comment, marked as answer]
This is no different to https://github.com/ppy/osu/issues/4201

**Reply by Doksha** [reply]
It absolutely is, this is referring to desktop (which should have completely different support anyways) that thread is really only referring to mobile.
Mobile is much smaller, meaning the way it should detect touches should reflect that, but with desktop, players are using much larger screens where it really doesnt make sense to even think of them the same

**Reply by smoogipoo** [reply]
There will be no difference in the way that the two detect touches, nor in the way that the touch areas will appear, nor in the behaviour of the two.
